12/umbraco-cms/content-delivery-api.md

```diff
@@ -0,0 +1,413 @@
+                "ApiKey": "my-api-key",
+                "DisallowedContentTypeAliases": ["alias1", "alias2", "alias3"]
+            }
+        }
+    }
+}
+```
+{% endcode %}
+
+* `Umbraco.CMS.DeliveryApi.PublicAccess` determines whether the Delivery API (_if enabled_) should be publicly accessible or if access should require an API key.
+* `Umbraco.CMS.DeliveryApi.ApiKey` specifies what API key to use for authorizing access to the API when public access is disabled. This setting is also used for managing access to draft content.
+* `Umbraco.CMS.DeliveryApi.DisallowedContentTypeAliases` contains the aliases of the content types that would never be exposed through the Delivery API.
+
+{% hint style="info" %}
+To test the functionality of the API, you would need to create some content items first.
+{% endhint %}
+
+## Endpoints
+
+The output produced by the Delivery API can either represent a specific content item or a paged list of multiple items. Before exploring the endpoints that API has to offer, there are a few concepts to keep in mind.
+
+### Concepts
+
+<details>
+
+<summary>Content item JSON Schema</summary>
+
+The Delivery API uses the JSON schema below to represent the retrieved content items, which consist of a range of properties. Basic JSON properties that any item returned by the API will contain include `name`, `id`, and t_ype_ (`contentType` in this case). All properties added to the content item can be found under the `properties` field.  Depending on the used property editor, the value associated with it in the JSON response can be a _string_, _number_, _boolean expression_, _array_, _object_ or _`null`_ (when a picker property is empty). The `route` property provides information about the `path` of the content item, as well as details about the root node value that is represented by the `startItem` object. We will discuss the concept of a `startItem` in more detail in the next section. Finally, if the content item varies by culture, the `cultures` property will contain information about all configured cultures for the content node, including the culture variant `path` and `startItem` for each one.
+
+```json
+{
+  "name": "string",
+  "route": {
+    "path": "string",
+    "startItem": {
+      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "path": "string"
+    }
+  },
+  "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+  "contentType": "string",
+  "properties": {
+    "property1Alias": "string",
+    "property2Alias": 0,
+    "property3Alias": true,
+    "property4Alias": [],
+    "property5Alias": {},
+    "property6Alias": null
+  },
+  "cultures": {
+    "cultureIsoCode1": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    },
+    "cultureIsoCode2": {
+      "path": "string",
+      "startItem": {
+        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+        "path": "string"
+      }
+    }
+  }
+}
```

Comment:
To make the point clearer, don't reuse the GUID between the start item and the actual item. They can of course be equal if one is requesting the root item, but... just to make a point 😄 

```suggestion
{
  "name": "string",
  "route": {
    "path": "string",
    "startItem": {
      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "path": "string"
    }
  },
  "id": "11fb598b-5c51-4d1a-8f2e-0c7594361d15",
  "contentType": "string",
  "properties": {
    "property1Alias": "string",
    "property2Alias": 0,
    "property3Alias": true,
    "property4Alias": [],
    "property5Alias": {},
    "property6Alias": null
  },
  "cultures": {
    "cultureIsoCode1": {
      "path": "string",
      "startItem": {
        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "path": "string"
      }
    },
    "cultureIsoCode2": {
      "path": "string",
      "startItem": {
        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
        "path": "string"
      }
    }
  }
}
```